### DISCUSSION TITLE
t.map is not a function

### DISCUSSION THREAD

**Discussion by happytolerance** [ZuodaoTech/everyone-can-use-english#1225] [Q&A]
![image](https://github.com/user-attachments/assets/74a3cef1-af40-48bc-9663-a49f0b40334e)
我打开后它就报t.map is not a function的错，macOS arm64 v0.7.4，

**Comment by an-lee** [comment]
一打开就报这个错？之前有用过吗？

**Reply by happytolerance** [reply]
我之前用过的，也用GitHub账号登陆过，但配置GPT那块的API可能配错了，后面再打开就只提示这个错，删除了直接下载最新版重新装还是报这个错

**Comment by happytolerance** [comment]
我之前用过的，也用GitHub账号登陆过，但配置GPT那块的API可能配错了，后面再打开就只提示这个错，删除了直接下载最新版重新装还是报这个错谢谢啦发自我的 iPhone在 2024年12月5日，17:11，an-lee ***@***.***> 写道：﻿
一打开就报这个错？之前有用过吗？

**Reply by an-lee** [reply]
可以删掉重新安装新版。删除的时候删干净一点，把 EnjoyLibary 文件夹都删了
